src/reducers/users.spec.tsx

```diff
@@ -0,0 +1,116 @@
+import reducer, {
+  actions,
+  getCurrentUser,
+  getUserById,
+  getUserByName,
+  initialState,
+} from './users';
+import { fakeUser } from '../helpers';
```

Comment:
I know that we are co-locating tests, but what about putting any test specific module in `src/__tests__`? That seems to be a standard location. Just keep in mind that `src/setupTests.tsx` is a magical path so it can't be moved, IIRC.

Reply:
When I moved the helper to `__tests__`, Jest now tries to run the file and I get the error: `Your test suite must contain at least one test`. I tried adding a `testPathIgnorePatterns` to `package.json`, but `create-react-app` complains that that is not allowed without ejecting.

What do you suggest?

Reply:
ok, then I suggest keeping the file where it is but renaming it to `testHelpers.js`

Reply:
... or even `test-helpers.tsx` ! I just saw that.